### DISCUSSION TITLE
Taylor factor calculation

### DISCUSSION THREAD

**Discussion by jerardg10** [mtex-toolbox/mtex#1053] [Ask Anything]
I am a little confused on how 'calcTaylor' is performing the Taylor factor calculation. Is it using the ratio of microscopic slip (numerator) to macroscopic von Mises equivalent strain (denominator) via a 'Full constraints' or 'Relaxed constraints' method? 

If you could also provide a literature reference for the particular method of choice that would be great. Thanks!

**Comment by hsheng94** [comment]
Hi, @jerardg10. I have the same confusion here. Have you got any response resolving this issue? 

Dr. @ralfHielscher, any update and comment on this post? If there is a reference from a paper or textbook that can explain the method here would be really helpful.

**Comment by stargazingdays** [comment]
Hi,

You can find the function calcTaylor.m under mtex-5.3\TensorAnalysis\@strainTensor. From what I can tell, the calculation seems to be following minimum slip kind of approach, and same strain is imposed.

I wonder if OIM uses the same approach? Or is it taking the sum of the reciprocals of Schmid factor? Any comments are greatly appreciated.

Best,
J

**Reply by stargazingdays** [reply]
Okay 5 activated slip systems shows up in OIM slip indicator. It makes me believe the algorithm is similar to CalcTaylor.
